procurement_plan/models/procurement.py

```diff
@@ -19,15 +19,38 @@ class ProcurementOrder(models.Model):
         ('production', 'Production'),
         ('transit', 'Transit Location')],
         string='Location Type', related="location_id.usage", store=True)
+    date_planned_copy = fields.Datetime(string='Scheduled Date')
 
     @api.model
     def create(self, data):
         if 'plan' in self.env.context and 'plan' not in data:
             data['plan'] = self.env.context.get('plan')
+        if 'date_planned' in data:
+            data['date_planned_copy'] = data.get('date_planned')
         procurement = super(ProcurementOrder, self).create(data)
         return procurement
 
     @api.multi
+    def write(self, values):
+        if 'date_planned' in values:
+            values['date_planned_copy'] = values.get('date_planned')
+        return super(ProcurementOrder, self).write(values)
+
+    @api.multi
+    @api.onchange('date_planned')
+    def _onchange_date_planned(self):
+        self.ensure_one()
```

Comment:
A ver si consigo explicarme porque no entiendo nada de lo que hablais.
La fecha del abastecimiento debe poder modificarse normalmente. Ahora bien... si ese abastecimiento tiene hijos... las fechas de los hijos se ven afectadas. Al igual que cuando tiene relacionados un pedido de compra o una of.
Solo en esos casos el campo podria ser no editable y su modificacion deberia ser hecha desde el wizard para no dejar inconsistencias.
En todo el resto de casos la fecha se podria cambiar sin ningun problema por tanto el campo es editable en la mayoria de los casos
Mi ultima propuesta fue dejarlo editable y dar un aviso de que usen el wizard para cambiarlo cuando se cumplen esas condiciones que provocan inconsistencia